app/src/main/res/layout/profile_progress_fragment.xml

```diff
@@ -13,37 +13,13 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent">
 
-    <com.google.android.material.appbar.AppBarLayout
-      android:id="@+id/profile_progress_app_bar_layout"
+    <FrameLayout
       android:layout_width="match_parent"
       android:layout_height="wrap_content"
-      android:theme="@style/ThemeOverlay.AppCompat.Dark.ActionBar"
       app:layout_constraintEnd_toEndOf="parent"
       app:layout_constraintStart_toStartOf="parent"
       app:layout_constraintTop_toTopOf="parent">
 
-      <androidx.appcompat.widget.Toolbar
-        android:id="@+id/profile_progress_toolbar"
-        android:layout_width="match_parent"
-        android:layout_height="?attr/actionBarSize"
-        android:background="?attr/colorPrimary"
-        android:fontFamily="sans-serif"
-        android:minHeight="?attr/actionBarSize"
-        android:textSize="20sp"
-        app:navigationContentDescription="@string/go_to_previous_page"
-        app:navigationIcon="?attr/homeAsUpIndicator"
-        app:title="@string/profile"
-        app:titleTextColor="@color/white" />
-    </com.google.android.material.appbar.AppBarLayout>
-
-    <FrameLayout
-      android:layout_width="0dp"
-      android:layout_height="0dp"
-      app:layout_constraintBottom_toBottomOf="parent"
-      app:layout_constraintEnd_toEndOf="parent"
-      app:layout_constraintStart_toStartOf="parent"
-      app:layout_constraintTop_toBottomOf="@id/profile_progress_app_bar_layout">
-
       <androidx.recyclerview.widget.RecyclerView
```

Comment:
Padding bottom in this recyclerview is different. 
## Portrait
<img width="452" alt="Screenshot 2020-04-14 at 11 46 38 AM" src="https://user-images.githubusercontent.com/9396084/79192086-ad168980-7e45-11ea-9438-8ad4be9a8ce2.png">

## Landscape
<img width="726" alt="Screenshot 2020-04-14 at 11 47 23 AM" src="https://user-images.githubusercontent.com/9396084/79192119-bf90c300-7e45-11ea-98d2-89bdd15b539f.png">

Reply:
Done